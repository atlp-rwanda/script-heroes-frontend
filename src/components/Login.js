import React from "react";

const Login = () => {
  return (
    <div>
      <h2>Sign In Component</h2>
    </div>
  );
};

export default Login;
