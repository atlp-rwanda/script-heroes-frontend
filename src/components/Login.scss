.login_image {
  width: 100%;
}
.login_form {
  background-color: #e1e1e1;
  border-radius: 10px;
}

.sub_title {
  text-align: center;
  color: #f9a826;
}
h1 {
  font-family: "Calibri" !important;
  text-align: center;
}
.small {
  width: 50px;
  height: 50px;
}

.form_part {
  padding-top: 10px;
  padding-bottom: 10px;
}
.social_link {
  text-decoration: none;
  background-color: #fff;
  padding: 10px;
  border: gray 1px solid;
  border-radius: 10px;
  margin: 10px;
}
.social_link:hover {
  text-decoration: none;
  border: #f9a826 1px solid;
  color: #f9a826;
}
.social_login {
  width: 30px;
  height: 30px;
  border-radius: 5px;
  margin-right: 10px;
  margin-top: -5px;
}

button {
  border-radius: 2px;
  border: 0;
  outline: none;
  padding: 5px;
  margin: 5px;
}

.container-login {
  margin-top: 20px;
  min-height: 25vh;
}
.login {
  background-color: #e9ecef;
  height: 400px;
  padding: 10px;
}
