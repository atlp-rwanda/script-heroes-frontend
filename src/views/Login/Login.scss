.login form {
  display: flex;
  justify-content: space-around;
  align-items: center;
  min-height: 10vh;
  max-width: 70vh;
  border: 3px solid #f1f1f1;
  text-align: center;
  margin-left: auto;
  margin-right: auto;
  padding-top: 30px;
  padding-bottom: 30px;

  input {
    width: 100%;
    padding: 12px 20px;
    margin: 8px 0;
    display: inline-block;
    border: 1px solid #ccc;
    box-sizing: border-box;
  }

  label {
    color: #728191;
  }

  button {
    background-color: #728191;
    color: white;
    padding: 14px 20px;
    margin: 8px 0;
    border: none;
    cursor: pointer;
    width: 100%;
  }
  button:hover {
    opacity: 0.8;
  }
}

.title {
  font-size: 40px;
  color: #f9a826;
  margin-bottom: 100px;
  margin-top: 30px;
  text-align: center;
  font-style: italic;
}

h2 {
  display: flex;
  justify-content: space-around;
  color: #f9a826;
  margin-bottom: 20px;
}
.container {
  padding: 16px;
}
