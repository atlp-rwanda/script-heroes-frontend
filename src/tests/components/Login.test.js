import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import Login from '../../components/Login';

describe('Footer', () => {
  it('Should match snapshot', () => {
    const wrapper = shallow(<Login />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
