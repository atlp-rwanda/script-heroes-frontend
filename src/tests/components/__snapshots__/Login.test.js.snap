// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Should match snapshot 1`] = `
<div>
  <div
    className="in"
  >
    <h2>
      Sign In Component
    </h2>
  </div>
</div>
`;
